Content.Server/Atmos/Rotting/RottingSystem.cs

```diff
@@ -157,6 +157,23 @@ private void OnTempIsRotting(EntityUid uid, TemperatureComponent component, ref
         args.Handled = component.CurrentTemperature > Atmospherics.T0C + 0.85f;
     }
 
+    /// <summary>
+    /// Is anything speeding up the decay?
+    /// e.g. buried in a grave
+    /// TODO: hot temperatures increase rot?
+    /// </summary>
+    /// <returns></returns>
+    private float GetRotRate(EntityUid uid)
+    {
+        if (_container.TryGetOuterContainer(uid, Transform(uid), out var container) &&
+            TryComp<ProRottingContainerComponent>(container.Owner, out var rotContainer))
+        {
+            return rotContainer.DecayModifier;
+        }
+
+        return 1f;
+    }
```

Comment:
Why doesn't this just check the immediate container and not the outermost container?

Reply:
True, I had just copied the code related to AntiRottingContainer. I had only been thinking about in the context of a grave, which couldn't go in other containers, but checking the immediate container would be better for component reuse. I will change this.